src/baguetteBox.js

```diff
@@ -598,6 +604,22 @@
         return show(currentIndex - 1);
     }
 
+    // Return false at the left end of the gallery
+    function showFirstImage(event) {
+        if (event) {
```

Comment:
Do we need the `if`? Is there a case where `event` is undefined?

Reply:
Currently no.
I included it in case you'd want to expose this function later on, cfr shownext etc.

Reply:
Could this be merged please? We could really use this feature...